web/themes/custom/pineapple/sass/layouts/views/_view--recipes.scss

```diff
@@ -0,0 +1,98 @@
+.view-recipes {
+
+  .views-exposed-form {
+    padding: 0;
+    margin-bottom: $unit *3;
+
+    .form--inline {
+      width:100%;
+      display: flex;
+      flex-direction: column;
+
+      @include breakpoint($lap) {
+        display: inline-flex;
+        flex-direction: row;
+        align-items: flex-end;
+      }
+
+      .form-item,
+      .form-actions {
+        margin: 0 12px 12px 0;
+        padding: 0;
+
+        @include breakpoint($lap) {
+          margin-bottom: 0;
+        }
+
+        &.form-item-title {
+          font-size: $small-font;
+          flex-grow: 1.5;
+        }
+
+        &.form-type-select {
+          flex-grow: 1.5;
+
+          @include breakpoint($lap) {
+            display: inline-flex;
+          }
+
+          .form-select {
+            padding: 0;
+            margin: 0;
+            margin-top: $unit/2;
+
+            @include breakpoint($lap) {
+              margin-top: 0;
+            }
+
+            &:hover {
+              background-color: $white;
+            }
+
+            option {
+              &:hover {
+                background-color: $gray;
+              }
+            }
+          }
+        }
+
+        &.form-composite {
+          flex-grow: 0.25;
+          .form-item {
+            display: inline-flex;
+            align-items: center;
+          }
+        }
+
+        input {
+          margin: 0;
+        }
+
+        label {
+          font-size: $small-font;
+          display: flex;
+          align-items: center;
+          margin: 0;
+
+          @include breakpoint($lap) {
+            margin: 0 $unit;
+          }
+        }
+      }
+
+      .form-actions {
+        flex-grow: 0.25;
+
+        .button {
```

Comment:
Currently when the apply button is clicked the filters are applied and the page reloads. The issue is that then this happens the page is scrolled back at the top even though you would’ve been lower down at the filters. You need to add a page redirect that will go to to the same page but would be scrolled back to a similar position. This can be done by appending an id to the end of the full url in JavaScript

Reply:
Done